duplicates-in-array.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function (array, n){
+  var hasDuplicates = false;
+  var sum = 0;
+  var length = array.length;
+    if ( array.length === 0 ) return false;
+  	if( n != length ) return false;
```

Comment:
The code indentation here makes it really hard to read.

Reply:
Yes, I see. Usually I am more careful with this. In my files  those two conditions are aligned.